### ISSUE TITLE
The Snackbar status variants are communicated through color alone

### ISSUE THREAD

**Issue by nicklemmon** [SparkPost/matchbox#234]
The `<Snackbar/>` component instances convey a different state meaning through color alone. For example:

![Screen Shot 2019-08-28 at 3 18 33 PM](https://user-images.githubusercontent.com/3613392/63886196-ef1eb800-c9a7-11e9-9ba2-5906861e2cdd.png)
![Screen Shot 2019-08-28 at 3 24 19 PM](https://user-images.githubusercontent.com/3613392/63886200-f219a880-c9a7-11e9-8452-e81c894ade0f.png)

In these two cases, a significantly different meaning is conveyed through the color of the component instance alone - which is unfortunately a WCAG violation: https://www.w3.org/TR/WCAG20-TECHS/G14.html

## Possible Solutions

- Incorporate an icon in to the `<Snackbar/>` component, leave current color treatments as-is
- Update the design of the `<Snackbar/>` to align more closely with the `<Banner/>`, i.e., using dark text on a light background, in addition to pairing with an icon. This could help resolve both this issue and #233 simultaneously
- Note that any icon included in the component instance would need to have meaningful alternate text for screen reader users to encounter

**Comment by jonambas** [comment]
Resolved in #238
